### PR TITLE
Add clientsets to gke-diagnoser tool

### PR DESCRIPTION
**Pull request by songrx1997**
Add general k8s clientsets
Add BackendConfig clientsets
Add FrontendCondif clientsets

### CODE REVIEW

File: cmd/gke-diagnoser/app/kube/clientset.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	beconfigclient "k8s.io/ingress-gce/pkg/backendconfig/client/clientset/versioned"
+	feconfigclient "k8s.io/ingress-gce/pkg/frontendconfig/client/clientset/versioned"
+)
+
+// NewClientSet returns a new Kubernetes clientset
+func NewClientSet(kubeContext, kubeConfig string) (*kubernetes.Clientset, error) {
```
Comment (bowei) [inline cmd/gke-diagnoser/app/kube/clientset.go:26]: kubeConfig => kubeConfigPath

this is a little confusing kubeConfig is path to the kubeConfig file.

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**